docs/_content/standaard/index.md

```diff
@@ -18,4 +18,6 @@ Hieronder de directe links naar alle Open API specificaties (OAS):
 * [Autorisaties API specificatie](https://ref.tst.vng.cloud/ac/api/v1/schema/)
 * [Notificaties API specificatie](https://ref.tst.vng.cloud/nrc/api/v1/schema/)
   * [Notificaties API specificatie voor consumers](https://rebilly.github.io/ReDoc/?url=https://ref.tst.vng.cloud/api-specificatie/nrc/consumer-api/openapi.yaml)
+  
```

Comment:
In de figuur hieronder is een overzicht weergegeven van de API's voor Zaakgericht werken. De Catalogi API en Zaken API zijn puur voor zaakgericht werken bedoeld, en hebben geen functie daarbuiten. De Documenten API en Besluiten API zullen ook buiten zaakgericht werken toepassingen krijgen. Immers, niet alle documenten zijn zaakgericht en niet alle besluiten zullen in de context van een zaak worden genomen. Denk bij het laatste bijv. aan raadsbesluiten. 

Naast deze API zijn er nog een aantal API’s ontwikkeld ter ondersteuning, t.w. Notificaties API, Autorisatie API en de Referentielijsten API.